c++/src/capnp/compat/json.capnp.h

```diff
@@ -5,9 +5,6 @@
 
 #include <capnp/generated-header-support.h>
 #include <kj/windows-sanity.h>
-#if !CAPNP_LITE
```

Comment:
Regenerated the header.

Reply:
How did you generate this header? When I run `regenerate-bootstraps.sh`, it brings this include back. It turns out this is due to a bug in the code generator which I'll fix: if a previous file listed on the same command line contained interfaces, then latter files have this include even if they don't need it.

But, are you generating this header using something other than the `regenerate-bootstraps.sh` script?

Reply:
TBH I used that script before but forgot that it existed. Based on zsh history, I just ran capnp_tool manually (`bazel-bin/src/capnp/capnp_tool compile --verbose -Isrc --no-standard-import --src-prefix=src -obazel-bin/src/capnp/capnpc-c++:src src/capnp/compat/json.capnp`) after noticing that the header had includes it didn't need.